vortex-layout/src/layouts/zoned/reader.rs

```diff
@@ -11,23 +11,22 @@ use dashmap::DashMap;
 use futures::future::{BoxFuture, Shared};
```

Comment:
I reverted this back to before the dynamic filter PR. This is handled in the scan/filter now